TombRaider1996/ProcessMemory.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System.Diagnostics;
+using System;
+
+namespace TR1
+{
+    /// <summary>
+    ///     The game version's original target platform.
+    /// </summary>
+    internal enum Platform
+    {
+        PC,
+        PSX
+    }
+
+    /// <summary>
+    ///     The supported process versions.
+    /// </summary>
+    internal enum ProcessVersion
+    {
+        ATI,
+        DOSBox,     // Steam's DOSBox
+        ePSXe180,
+        ePSXe190,
+        ePSXe1925,
+        ePSXe200
+    }
+
+    /// <summary>
+    ///     The memory sizes of supported process versions (in bytes).
+    /// </summary>
+    internal enum ExpectedSize
+    {
+        ATI = 3092480,
+        DOSBox = 40321024,
+        ePSXe180 = 10231808,
+        ePSXe190 = 10301440,
+        ePSXe1925 = 10518528,
+        ePSXe200 = 20287488
+    }
+
+    /// <summary>
+    ///     The supported PSX versions of the game.
+    /// </summary>
+    internal enum PSXGameVersion
+    {
+        USA_1_0,
+        USA_final,
+        EU,
+        FR,
+        GER,
+        JP
+    }
+
+    /// <summary>
+    ///     Manages the process's watched memory values for <see cref="Autosplitter"/>'s use.
+    /// </summary>
+    internal class ProcessMemory
+    {
+        private Process _process;
+        public GameData gameData;
```

Comment:
Nitpick: if following [MS guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers), this should be Pascal case since it's public.
```suggestion
        public GameData GameData;
```

Reply:
Done.